pyembed/src/interpreter_config.rs

```diff
@@ -6,12 +6,13 @@
 
 use {
     crate::{config::ResolvedOxidizedPythonInterpreterConfig, NewInterpreterError},
-    libc::{c_int, size_t, wchar_t},
+    core::{usize,i32,u16},
```

Comment:
Clippy says `u16` is unused outside of Windows.

So we need to import any of these primitives anyway? They are part of the standard library prelude and should always exist.

Reply:
Very true, I'll remove the import statements